content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -9,22 +9,29 @@ weight: 70
 
 {{% capture overview %}}
 
-{{< feature-state state="alpha" >}}
+{{< feature-state state="beta" >}}
```

Comment:
It looks like you removed some of my copy-edit, such as the feature-state indicating that the feature was alpha in 1.8 and is now beta in 1.11. It is useful for people to be able to see that, since they need to make a decision about which version to run. It's also good not to mention version-specific stuff in the body of the text when we can avoid it by using the feature-state tags, since they are easier to search and replace later on. Mentioning version-specific stuff in the body text leads to situations where we have docs specifically mentioning versions of K8s that are no longer supported.

If you insist on having the info broken out into stuff about 1.8, 1.9, 1.10, and 1.11 separately (as below), consider using a table to outline the evolution of the behavior through versions, rather than expecting people to parse it out of multiple paragraphs.

Reply:
I changed part of the wording, because it was repeating the same thing mentioned a line above. Anyway, I rephrased and added a table. PTAL.